pkgs/by-name/lu/lugaru/package.nix

```diff
@@ -52,6 +52,14 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DSYSTEM_INSTALL=ON" ];
 
+  postPatch = ''
+    substituteInPlace CMakeLists.txt --replace-fail \
+      "cmake_minimum_required(VERSION 3.0)" \
+      "cmake_minimum_required(VERSION 3.5)" \
```

Comment:
Would it be possible to bump up to 3.10, while it nice to get it working with cmake 3.5, it mean this might fail again in not so long. This is what is recommended in the tracking pr